.github/workflows/pr-assessment.yaml

```
@@ -0,0 +1,14 @@
+name: GitHub PR Size Assessment
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)